packages/backend/src/fee/service.ts

```diff
@@ -29,41 +30,52 @@ export async function createFeeService({
     knex
   }
   return {
-    create: (options: CreateOptions) => createFee(deps, options)
+    create: (options: CreateOptions) => createFee(deps, options),
+    getLatestFee: (assetId: string) => getLatestFee(deps, assetId)
   }
 }
 
+async function getLatestFee(
+  deps: ServiceDependencies,
+  assetId: string
+): Promise<Fee | undefined> {
+  return await Fee.query(deps.knex)
+    .where('assetId', assetId)
+    .whereNotNull('activatedAt')
+    .orderBy('activatedAt', 'desc')
+    .first()
+}
+
 async function createFee(
   deps: ServiceDependencies,
-  options: CreateOptions
+  { assetId, type, fee }: CreateOptions
 ): Promise<Fee | FeeError> {
-  try {
-    const fee: Partial<Fee> = {
-      assetId: options.assetId,
-      type: options.type
-    }
+  const { fixed, percentage } = fee
 
-    if (options.fee.percentage !== undefined) {
-      fee.percentageFee = options.fee.percentage.toString()
-    }
+  if (fixed === BigInt(0) && Number(percentage.toFixed(4)) === 0) {
```

Comment:
I suppose that makes more sense - I was thinking no fee record would be the way to represent no fees but a fee with 0 for both would allow the ASE to update from some fee to no fee.